DTS-STN/Service-Canada-Labs
# SCDS Footer & HelpIcon

## [Use SCDS Footer](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=133820)

## [Update help/information component in ProjectInfo section of Project pages](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=130274)

This PR includes the implementation of the SCDS Footer component and the SCDS HelpIcon component. Our own footer in the layout has been replaced with the SCDS component. On project pages, in the ProjectInfo component, the HelpIcon has replaced the previous tooltip implementation.

### Footer

Before:
![Screenshot 2023-08-15 at 8 49 07 AM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/f27f8297-44cc-4f1d-9a90-f45e2c50d9ba)

After:
![Screenshot 2023-08-15 at 8 49 22 AM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/8134bbba-fdfa-45a6-a8d6-267b057cf900)

### Tooltip/HelpIcon

Before:
![Screenshot 2023-08-15 at 8 48 32 AM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/ed1500ae-1c1c-4f93-80e6-ae1255b00005)

After:
![Screenshot 2023-08-15 at 8 48 17 AM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/097c577a-9301-4a4a-8f4f-ca180a44104e)


## Test Instructions

1. Go to any number of pages
2. See that Footer renders properly, is responsive and links work
3. Go to project pages
4. Select the information icon and see that it displays the modal with the proper information
